src/cmdline/mod.rs

```diff
@@ -281,6 +281,16 @@ pub struct CommonCustomizeConfig {
     /// including when Ignition is run.
     #[clap(long, value_name = "path")]
     pub network_keyfile: Vec<String>,
+    /// Nmstate file to generate NetworkManager keyfile for live & dest
+    ///
+    /// Configure networking using the generated NetworkManager keyfile.
+    /// Network settings will be applied in the live environment, including
+    /// when Ignition is run.  If installer is enabled via additional options,
+    /// network settings will also be applied in the destination system,
+    /// including when Ignition is run.
+    #[clap(long, value_name = "path")]
+    pub network_nmstate: Option<String>,
```

Comment:
For consistency with `--network-keyfile`, let's make this be a `Vec<String>` too instead?

Reply:
Done